.babelrc

```diff
@@ -0,0 +1 @@
+{}
```

Comment:
Let's not merge until we can remove this. This is only necessary until a new release of Jest is cut. See https://github.com/danger/danger-js/issues/261